src/samplers/syscall/latency/mod.bpf.c

```diff
@@ -76,6 +116,22 @@ int sys_exit(struct trace_event_raw_sys_exit *args)
 		__sync_fetch_and_add(cnt, 1);
 	}
 
+	// for some syscalls, we track counts by "family" of syscall. check the
+	// lookup table and increment the appropriate counter
+	idx = 0;
+	if (syscall_id < 336) {
+		u32 counter_idx = syscall_lookup[syscall_id];
+
+		if (counter_idx < 8) {
```

Comment:
Agree on moving the grouping numbers up. I think we still want to do a bounds check on the counter_idx after the lookup. This gives us a guarantee that the LUT data and counter grouping are in agreement and avoids accidentally writing into another CPU's counter group.

Reply:
Yes, I think a check is good to avoid accidental memory corruption. This defensive pattern is something I picked up while working on twemcache- we used to put 'NOT_REACHED()', which is a macro, on branches that should never be taken, and always complete the if/else statement to make it symmetrical